index.php

```diff
@@ -14,6 +14,14 @@
 }
 
 try {
+    foreach ($request as $value) {
+        if (! is_string($value)) {
+            http_response_code(400);
+            print render('error.html.php', ['message' => '400 Bad Request']);
```

Comment:
Maybe be more specific:

```suggestion
    foreach ($request as $key => $value) {
        if (! is_string($value)) {
            http_response_code(400);
            print render('error.html.php', [
              'title' => '400 Bad Request',
              'message' => "Query string parameter “{$key}” is not a string.",
            ]);
```